client/src/containers/Annuaire/AnnuaireLecture/AnnuaireLecture.component.tsx

```diff
@@ -145,6 +145,8 @@ export const AnnuaireLectureComponent = (props: Props) => {
           t={props.t}
           letterSelected={letterSelected}
           setLetterSelected={setLetterSelected}
+          setFilteredStructures={setFilteredStructures}
+          sortStructureByAlpha={sortStructureByAlpha}
```

Comment:
javais pas fait attention à ca hier, mais le nom sortStructureByAlpha est pas tres heureux car on pense que cest une fonction qui te sort les structure. Alors que cest un tableau de structures. ce serait mieux de le nommer sortedStructuresByAplha

Reply:
et aussi par cohérence ce serait mieux que ca vaille [] plutot que nul quand filterStructure est falsy ( by the way filterSttructure est pourri comme nom aussi on croit aussi que cest une fonction, ca devrait etre filteredStructures)